.github/workflows/build.yml

```diff
@@ -33,6 +33,30 @@ jobs:
         env:
           DOTNET_CLI_TELEMETRY_OPTOUT: true
         run: dotnet publish --configuration Release --self-contained --runtime linux-x64 Cmf.CustomerPortal.Sdk.sln
+
+      # Run tests with coverage
+      - name: Run Unit Tests with Coverage
+        run: |
+          dotnet test Cmf.CustomerPortal.Sdk.sln --configuration Release \
+            --collect:"XPlat Code Coverage" \
+            --logger trx \
+            --results-directory TestResults
+
+      # Generate coverage report
+      - name: Generate Coverage Report
```

Comment:
I don't think we should generate the HTML code coverage artifact. The other extensions relies on the cobertura files.